src/Shiny.Notifications/Platforms/Android/NotificationManager.cs

```diff
@@ -253,11 +282,11 @@ protected virtual void AddCategory(NotificationCompat.Builder builder, Notificat
             var category = this.registeredCategories.FirstOrDefault(x => x.Identifier.Equals(notification.Category));
             if (category == null)
             {
-                Log.Write("Notifications", "No notification category found for " + notification.Category);
+                Log.Write(NotificationLogCategory.Notifications, "No notification category found for " + notification.Category);
```

Comment:
To understand why this is different than the resourceId exceptions above, it is because the user can change there categories from one day to the next and all scheduled notifications would start to fail.  Categories are not as "static" as resources and general purpose args